doc/cygnus-ngsi/installation_and_administration_guide/name_mappings.md

```diff
@@ -0,0 +1,75 @@
+                     "originalEntityId": "Room1",
+                     "originalEntityType": "Room",
+                     "newEntityId": "new_room1",
+                     "newEntityType": "new_room",
+                     "attributeMappings": [
+                        {
+                           "originalAttributeName": "temperature",
+                           "originalAttributeType": "centigrade",
+                           "newAttributeName": "new_temp",
+                           "newAttributeType": "new_cent"
+                        },
+                        ...
+                     ]
+                  },
+                  ...
+               ]
+            },
+            ...
+         ]
+      },
+      ...
+   ]
+}
+```
+
+When a notification is sent to Cygnus, a special Flume interceptor called `NGSINameMappingsInterceptor` intercepts the plain `Event`'s generated by `NGSIRestHandler`, parses the `Event`'s body and iterate on the configured mappings in order to create a mapped version of the original notification. Once finished, both versions of the notification, original and mapped ones, are put into the channel buy means of a `NGSIEvent`, a Java object able to handle:
+
+* The original headers sent by `NGSIRestHandler`, i.e. `fiware-service`, `fiware-servicepath`, `fiware-correlator` and `transaction-id`.
+* New headers regarding the mapped FIWARE service and FIWARE service path, i.e. `mapped-fiware-service` and `mapped-fiware-service-path`.
+* The original notification already parsed as a `NotifyContextRequest` object.
+* The mapped version of the original notification as a `NotifyContextRequest` object.
+
+Please observe no raw bytes about the body are sent.
+
+Whenever a sink takes one of these `NGSIEvent`'s, it is only a matter of deciding if such a sink enables the name mappings (`enable_name_mappings` parameter) or not. If name mappings are enabled, then the already parsed `NotifyContextRequest`, mapped version, is used. If not, then the original version is used.
+
+##Important notes
+Please observe the name mappings definition is global to all the sinks, at `NGSIRestHandler`, but then the application is local to the sink, depending on the `enable_name_mappings` parameter. Thus, if any of yur sinks is going to take advantage of the mappings, simply leave blank the mappings configuration file in the REST handler. That will avoid unncessary interception and iterations on the mappings.
+
+Additionally, please observe if any of the original names is not present, then the mapping affects all the names of that type. For instance, if `originalService` is not present in the fist mapping, then the mapping affects all the FIWARE services.
+
+[Top](#top)
+
+##Name mappings vs. grouping rules
+As seen, the name mappings feature is quite similar to the already existent grouping rules. Both of them are Flume interceptors and bot h of them allow chaning certain notified name elements. Thus, which are the differences? Mainly:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1198/commits/3f5e44ab8c66588e5fa37a684e128ebd50852f79